UnityProject/Assets/Scripts/Furniture/OccupiableDirectionalSprite.cs

```diff
@@ -75,7 +84,7 @@ public void Awake()
 	private void OnWillDestroyServer(DestructionInfo info)
 	{
 		//release the player
-		if (occupant > 0)
+		if (occupantNetId > 0)
```

Comment:
Will be preferable to use the NetId.Empty constant instead of 0 (makes it more pretty)